### PR TITLE
Change user list style

### PR DESCRIPTION
**Pull request by TruddyTheDuddi**

### CODE REVIEW

File: app/src/main/res/layout/profile_row_item.xml [hunk 1]
```diff
@@ -5,87 +5,101 @@
     android:id="@+id/frameLayout2"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:padding="8dp">
+    android:paddingVertical="5dp">
```
Comment (Gerussi) [inline app/src/main/res/layout/profile_row_item.xml:8]: Should we at some point unify the padding throughout the app and store those in dimens ?

File: app/src/main/res/values/style.xml [hunk 2]
```diff
@@ -6,6 +6,12 @@
         <item name="color">@color/theme_bright</item>
     </style>
 
+    <style name="profileListCirclePic" parent="">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerSize">30dp</item>
+        <item name="color">@color/theme_bright</item>
```
Comment (Gerussi) [inline app/src/main/res/values/style.xml:12]: Did you look if you could use an ?attr ?

**Review by Gerussi — APPROVED** [review]

File: app/src/main/res/values/style.xml (hunk 2 above)
Comment (TruddyTheDuddi) [reply app/src/main/res/values/style.xml:12]: No but it is not too important here

File: app/src/main/res/layout/fragment_profile_list.xml [hunk 3]
```diff
@@ -5,11 +5,28 @@
...
+        android:paddingHorizontal="20dp"
+        android:textSize="14sp" />
+
     <SearchView
         android:id="@+id/profile_list_search"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" >
-
+        android:layout_height="wrap_content"
+        android:queryHint="Search for user">
```
Comment (atappy) [inline app/src/main/res/layout/fragment_profile_list.xml:29]: would it be possible to move this hint string to the strings so we can change the language ? That would be a wonderful add for foreign users, isn't it ?

**Review by atappy — APPROVED** [review]
good job !

File: app/src/main/res/layout/fragment_profile_list.xml (hunk 3 above)
Comment (TruddyTheDuddi) [reply app/src/main/res/layout/fragment_profile_list.xml:29]: Sorry my bad, it was in strings I just didn't link it

**Pull request merged**